File: 백준/Bronze/11005. 진법 변환 2/진법 변환 2.cc
#include <iostream>

using namespace std;

int main(void) {
    int num, base;
    int i;
    int converted[10000000];
    
    cin >> num >> base;
    for (i = 0; num / base != 0; i++) {
        converted[i] = num % base;
        num /= base;
    }
    converted[i] = num;
    
    for (int j = i; j >= 0; j--) {
        if (converted[j] < 10) 
            cout << converted[j];
        else
            cout << (char) (converted[j] - 10 + 'A');
    }
    cout << "\n";
    return 0;
}

File: 백준/Bronze/11005. 진법 변환 2/진법 변환 2.py
num, base = map(int, input().split())

conversion_number = ""
while (num != 0) :
    digit = num % base
    if (0 <= digit <= 9) :
        conversion_number = str(digit) + conversion_number
    else :
        conversion_number = chr(ord('A') + digit - 10) + conversion_number
    num //= base
    
print(conversion_number)